tox.ini

```diff
@@ -21,9 +20,10 @@ deps =
     django19: Django >= 1.9, < 1.10
     django110: Django >= 1.10, < 1.11
     django111: Django >= 1.11, < 1.12
-    django-dev: https://github.com/django/django/tarball/master
+    djangodev: https://github.com/django/django/tarball/master
 commands=
-    python runtests.py
+    coverage run --source polymorphic runtests.py
+    coverage xml -i
```

Comment:
Why generate the XML here? AFAIK, codecov doesn't need that explicitly (or am I missing something?).